iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -155,6 +155,36 @@ public void after() throws Exception {
     HiveIcebergStorageHandlerTestUtils.close(shell);
   }
 
+  @Test
+  public void testPartitionTransform() {
+    Schema schema = new Schema(
+        optional(1, "id", Types.LongType.get()),
+        optional(2, "year_field", Types.DateType.get()),
+        optional(3, "month_field", Types.TimestampType.withZone()),
+        optional(4, "day_field", Types.TimestampType.withoutZone()),
+        optional(5, "hour_field", Types.TimestampType.withoutZone()),
+        optional(6, "truncate_field", Types.StringType.get()),
+        optional(7, "bucket_field", Types.StringType.get()),
+        optional(8, "identity_field", Types.StringType.get())
+    );
+    PartitionSpec spec = PartitionSpec.builderFor(schema).year("year_field").month("month_field").day("day_field")
```

Comment:
nit: newline before, and I think it is more readable if we break after every parameter, like:
```
PartitionSpec.builderFor(schema)
.year("year_field")
.month("month_field")
.day("day_field")
.hour("hour_field")
.truncate("truncate_field", 2)
.bucket("bucket_field", 2)
.identity("identity_field")
.build();
```

Reply:
Done